docs/ref/pgcopydb_clone.rst

```diff
@@ -903,81 +903,54 @@ Examples
-   16738 | public |            actor |         399ms |            7999 B |       2 |                 116ms
-   16748 | public |         category |         170ms |             526 B |       1 |                 200ms
-   16805 | public |          country |          63ms |            3918 B |       1 |                 226ms
-   16900 | public |            staff |         170ms |             272 B |       1 |                 114ms
-   16832 | public |         language |         115ms |             276 B |       1 |                  68ms
-   16911 | public |            store |          88ms |              58 B |       2 |                 185ms
+   08:13:13.961 42893 INFO   [SOURCE] Copying database from "postgres://pagila:0wn3d@source/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
+   08:13:13.961 42893 INFO   [TARGET] Copying database into "postgres://pagila:0wn3d@target/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
+   08:13:14.009 42893 INFO   Using work dir "/tmp/pgcopydb"
+   08:13:14.017 42893 INFO   Exported snapshot "00000003-000000EB-1" from the source database
+   08:13:14.019 42904 INFO   STEP 1: fetch source database tables, indexes, and sequences
+   08:13:14.339 42904 INFO   Fetched information for 5 tables (including 0 tables split in 0 partitions total), with an estimated total of 1000 thousands tuples and 128 MB on-disk
+   08:13:14.342 42904 INFO   Fetched information for 4 indexes (supporting 4 constraints)
+   08:13:14.343 42904 INFO   Fetching information for 1 sequences
+   08:13:14.353 42904 INFO   Fetched information for 1 extensions
+   08:13:14.436 42904 INFO   Found 1 indexes (supporting 1 constraints) in the target database
+   08:13:14.443 42904 INFO   STEP 2: dump the source database schema (pre/post data)
+   08:13:14.448 42904 INFO    /usr/bin/pg_dump -Fc --snapshot 00000003-000000EB-1 --section=pre-data --section=post-data --file /tmp/pgcopydb/schema/schema.dump 'postgres://pagila:0wn3d@source/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60'
+   08:13:14.513 42904 INFO   STEP 3: restore the pre-data section to the target database
+   08:13:14.524 42904 INFO    /usr/bin/pg_restore --dbname 'postgres://pagila:0wn3d@target/pagila?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60' --section pre-data --jobs 2 --use-list /tmp/pgcopydb/schema/pre-filtered.list /tmp/pgcopydb/schema/schema.dump
+   08:13:14.608 42919 INFO   STEP 4: starting 8 table-data COPY processes
+   08:13:14.678 42921 INFO   STEP 8: starting 8 VACUUM processes
+   08:13:14.678 42904 INFO   Skipping large objects: none found.
+   08:13:14.693 42920 INFO   STEP 6: starting 2 CREATE INDEX processes
+   08:13:14.693 42920 INFO   STEP 7: constraints are built by the CREATE INDEX processes
+   2024-06-10 08:13:14.695 +03 [42955] LOG:  unexpected EOF on client connection with an open transaction
```

Comment:
That looks like you're using debug level Postgres settings that pushes server logs as notifications on the client system, and I would rather we do not have that in our example here. That's just a distraction for the docs.

Also, if the server side log is due to pgcopydb operations, we will need to fix that in another PR.

Reply:
Removed.